bitmovin/analytics/queries.js

```diff
@@ -1,57 +1,18 @@
 import urljoin from 'url-join';
 import http, { utils } from '../utils/http';
-import { queryBuilder } from './query_builder';
+import queryBuilder from './query_builder';
 
 export const queries = (configuration, http) => {
   const { get, post } = http;
   const baseUrl = urljoin(configuration.apiBaseUrl, 'analytics/queries');
 
-  const fn = (queryId) => {
-    return {};
-  };
+  const queryMethods = ['count', 'sum', 'avg', 'min', 'max', 'median', 'percentile', 'stddev', 'variance']
+    .reduce((obj, method) =>
+      ({ ...obj, [method]: (query) => post(configuration, urljoin(baseUrl, method), query) }), {});
 
-  fn.count = (query) => {
-    const url = urljoin(baseUrl, 'count');
-    return post(configuration, url, query);
-  };
+  const fn = Object.assign(() => ({}), queryMethods);
```

Comment:
@Tigraine do you know why the return value needs to be an object? I left it this way because it was like this before and it's also tested for (tests/bitmovin.test.js:30), but I cannot think of a good reason why we need this.

Reply:
No reason..